bin/index.js

```diff
@@ -17,7 +17,22 @@ var setDNSLib = require('../lib')({
 if (argv['test']) {
   setDNSLib.testAccountKey(argv['test'])
 } else if (argv['d'] && argv['p'] && argv['r']) {
-  setDNSLib.setDNS(argv['d'], argv['r'], argv['p'])
+  http.get({
+    host: 'ipfs.io',
+    port: 80,
+    path: argv['p']
+  }, function (res) {
+    if (res.statusCode === 302) {
```

Comment:
I'd say we can treat everything in the 2xx/3xx range as success. If we can't at the moment because of any weird gateway behaviour, we should make changes to the gateway.